editor/components/index.js

```diff
@@ -0,0 +1,38 @@
+// Post Related Components
+export { default as AutosaveMonitor } from './autosave-monitor';
+export { default as DocumentOutline } from './document-outline';
+export { default as MetaBoxes } from './meta-boxes';
+export { default as PageAttributes } from './page-attributes';
+export { default as PageAttributesCheck } from './page-attributes/check';
+export { default as PostAuthor } from './post-author';
+export { default as PostAuthorCheck } from './post-author/check';
+export { default as PostComments } from './post-comments';
+export { default as PostExcerpt } from './post-excerpt';
+export { default as PostFeaturedImage } from './post-featured-image';
+export { default as PostFormat } from './post-format';
+export { default as PostFormatCheck } from './post-format/check';
+export { default as PostLastRevision } from './post-last-revision';
+export { default as PostLastRevisionCheck } from './post-last-revision/check';
+export { default as PostPendingStatus } from './post-pending-status';
+export { default as PostPendingStatusCheck } from './post-pending-status/check';
+export { default as PostPingbacks } from './post-pingbacks';
+export { default as PostPreviewButton } from './post-preview-button';
+export { default as PostPublishButton } from './post-publish-button';
+export { default as PostPublishButtonLabel } from './post-publish-button/label';
+export { default as PostSavedState } from './post-saved-state';
+export { default as PostSchedule } from './post-schedule';
+export { default as PostScheduleLabel } from './post-schedule/label';
+export { default as PostSticky } from './post-sticky';
+export { default as PostStickyCheck } from './post-sticky/check';
+export { default as PostTaxonomies } from './post-taxonomies';
+export { default as PostTrash } from './post-trash';
+export { default as PostVisibility } from './post-visibility';
+export { default as PostVisibilityLabel } from './post-visibility/label';
+export { default as UnsavedChangesWarning } from './unsaved-changes-warning';
```

Comment:
It seems like some components that depend on the post state could have `Post` prefix, too. `PageAttributes` makes it harder to come up with a good pattern here :)

Reply:
Agreed, did this change for one or two components but gave up, we could do it separately one by one.

Reply:
I feel your pain, it can be combined with `README` task. We would have to leave an additional note when applicable.